src/Sylius/Component/Product/Model/ProductVariantInterface.php

```diff
@@ -45,7 +45,7 @@ public function addOptionValue(ProductOptionValueInterface $optionValue): void;
 
     public function removeOptionValue(ProductOptionValueInterface $optionValue): void;
 
-    public function hasOptionValue(ProductOptionValueInterface $optionValue): bool;
+    public function hasOptionValue(?ProductOptionValueInterface $optionValue): bool;
```

Comment:
I wonder if this change is not a BC Break, although it expands the accepted argument, wdyt @lchrusciel

Reply:
🚨 That's a BC break, all classes implementing this interface have to change their method signature.